runtime/calamari/src/lib.rs

```diff
@@ -173,12 +173,20 @@ parameter_types! {
     pub const SS58Prefix: u8 = manta_primitives::constants::CALAMARI_SS58PREFIX;
 }
 
+parameter_types! {
+    pub UnpausablePallets: Vec<Vec<u8>> = vec![b"Democracy".to_vec(), b"Balances".to_vec(), b"Council".to_vec(), b"CouncilCollective".to_vec(), b"TechnicalCommittee".to_vec(), b"TechnicalCollective".to_vec()];
+}
+
 impl pallet_tx_pause::Config for Runtime {
     type Event = Event;
-    type UpdateOrigin = EitherOfDiverse<
+    type Call = Call;
+    type MaxCallNames = ConstU32<10>;
```

Comment:
Pallet Asset is the pallet with the most calls at 25 looking at the metadata...  25 maybe a reasonable parameter to set here for that reason as @ghzlatarev mentioned in top comment.

We could address this with a maintenance mode later, but we would have to pause calls for `pallet_assets` individually right now.